tests/test_comm.rs

```diff
@@ -1,105 +1,82 @@
 use common::util::*;
-use std::ffi::OsStr;
 
 static UTIL_NAME: &'static str = "comm";
-fn at_and_ucmd() -> (AtPath, UCommand) {
-    let ts = TestScenario::new(UTIL_NAME);
-    let ucmd = ts.ucmd();
-    (ts.fixtures, ucmd)
-}
 fn new_ucmd() -> UCommand {
     TestScenario::new(UTIL_NAME).ucmd()
 }
```

Comment:
I believe the same definition of `fn new_ucmd` in each test can be replaced by a macro.